### ISSUE TITLE
Fix goreleaser builds 

### ISSUE THREAD

**Issue by onedr0p** [onedr0p/exportarr#124]
# Details

The latest release had an issue building on Darwin/arm64

**Comment by rtrox** [comment]
Looking at the output, I think all platforms would have failed, darwin just happened to get there first. [it looks like `url.JoinPath` was added in Go 1.19](https://pkg.go.dev/net/url#JoinPath) (which surprises me, I feel like I've been using it longer than that). Our goreleaser workflow was pinned to use 1.18, so that method wasn't available yet.

In the PR I opened, I unpinned the go version, I figure since this is a binary release, we don't need to worry about supporting older versions of go. So it should now download the latest go version like the `go-tests` job does.

I mentioned this in the PR, but I hadn't seen the issue open, so I'll repeat it here -- I don't actually know a good way to get goreleaser to upload artifacts to an existing release. in the past, I've had to delete the tag in github, delete the release, and `git push --tags` again to retry the goreleaser job. If there's a better way to do it, I'd love for you to share, though, my way feels a little scorched earth
